device/iot-device-samples/multiplexing-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/MultiplexingClientManager.java

```diff
@@ -53,6 +53,7 @@
      * @param multiplexingClient the multiplexing client
      * @param multiplexClientId user defined Id for the multiplexing client.
      */
+    @SuppressWarnings("SameParameterValue") // For the purpose of this sample, multiplexClientId is "MultiplexingClient", but it can be assigned to any user defined value.
```

Comment:
Aren't there tests for this method?
If there are - we can change id field to something else and the warning will go away

Reply:
this is a sample, so no, no tests for this.